src/main/java/org/liliya/hotelapp/configuration/Configuration.java

```diff
@@ -5,33 +5,52 @@
 import java.util.Properties;
 
 public class Configuration {
+    private static Configuration instance;
     private boolean statusChangeAvailability;
-    private String filePath;
+    private String dbUrl;
+    private String dbUsername;
+    private String dbPassword;
 
-    public Configuration() {
+    private Configuration() {
         loadProperties();
     }
 
+    public static Configuration getInstance() {
+        if (instance == null) {
+            instance = new Configuration();
+        }
+        return instance;
+    }
+
```

Comment:
Singlton ?
👍

Reply:
yes, I used it for creating configuration instance once